extensions/dub-link-shortener/src/shorten-link.tsx

```diff
@@ -69,67 +69,50 @@ const ShortenLinkForm = ({ retryValues, args }: { retryValues?: ShortLinkFormVal
     validation: {
       url: FormValidation.Required,
       domain: FormValidation.Required,
-      workspaceId: !isSingleWorkspace ? FormValidation.Required : undefined,
     },
     initialValues: retryValues ?? { domain: "dub.sh", url: isEmpty(url) ? originalLink : url, key },
   });
 
   return (
     <Form
       searchBarAccessory={<Form.LinkAccessory text="Dashboard" target="https://app.dub.co" />}
-      isLoading={isLoading || isLoadingLink}
+      isLoading={isLoadingLink}
       actions={
         <ActionPanel>
           <Action.SubmitForm title="Shorten Link" icon={Icon.Link} onSubmit={handleSubmit} />
         </ActionPanel>
       }
     >
       <Form.Description title={"❗"} text="Your shortened link will be copied to your clipboard." />
-      {!isSingleWorkspace && !error && (
-        <Form.Dropdown {...itemProps.workspaceId} title="Workspace">
-          {workspaces.map((w) => (
-            <Form.Dropdown.Item
-              key={w.id}
-              value={w.id}
-              title={w.name}
-              icon={{ source: w.logo ?? "command-icon.png", mask: Image.Mask.Circle }}
-            />
-          ))}
-        </Form.Dropdown>
-      )}
       <Form.Separator />
       <Form.TextField {...itemProps.url} placeholder="https://dub.co" title="Original URL" />
       <Form.TextField {...itemProps.key} placeholder="(Optional)" title="URL Key" />
       {/* todo: Add commands to create/manage tags in the workspace */}
-      <Form.TagPicker {...itemProps.tagIds} title="Tags">
-        {workspaces
-          .find((w) => w.id === values.workspaceId)
-          ?.tags?.map((t) => (
-            <Form.TagPicker.Item
-              key={t.id}
-              value={t.id}
-              title={t.name}
-              icon={{ source: Icon.Tag, tintColor: Color.Orange }}
-            />
-          ))}
+      <Form.TagPicker {...itemProps.tagIds} title="Tags" placeholder="(Optional)">
+        {tags.map((t) => (
+          <Form.TagPicker.Item
+            key={t.id}
+            value={t.id}
+            title={t.name}
+            icon={{ source: Icon.Tag, tintColor: Color.Orange }}
+          />
+        ))}
       </Form.TagPicker>
       <Form.Dropdown {...itemProps.domain} title="Domain">
         <Form.Dropdown.Item
           key="dub.sh"
           value="dub.sh"
           title="dub.sh"
-          icon={{ source: Icon.Link, tintColor: Color.Blue }}
+          icon={{ source: Icon.Globe, tintColor: Color.Blue }}
```

Comment:
Question (non-blocking): Should this color also be Purple like other domains?

Reply:
Trying to distinguish between [default Dub domains](https://dub.co/help/article/default-dub-domains) and [custom domains](https://dub.co/help/article/how-to-add-custom-domain), hence the color difference 😄